tests/net/deckard/client.rs

```diff
@@ -26,11 +32,19 @@ pub async fn do_client<R: SendRequest<RequestMessage<Vec<u8>>>>(
             StepType::CheckAnswer => {
                 let answer = resp.take().unwrap();
                 if !match_msg(step.entry.as_ref().unwrap(), &answer, true) {
+                    println!(
+                        "Reply message does not match at step {}",
+                        step_value.get()
+                    );
                     panic!("reply failed");
                 }
             }
-            StepType::TimePasses
-            | StepType::Traffic
+            StepType::TimePasses => {
+                clock.adjust_time(Duration::from_secs(
```

Comment:
See earlier comments on this PR about using `tokio::time::advance()` here instead.